README.md

```diff
@@ -53,7 +53,7 @@ Then installation is as simple as
 ```bash
 kubectl krew install konfig
 ```
-The plugin will be available as `kubectl konfig`, see [doc/USAGE](doc/USAGE.md) for further details.
+The plugin will be available as `kubectl konfig`, see [doc/USAGE](doc/USAGE.md) for further details. You could also define an alias as well: `alias konfig = 'kubectl konfig'`
```

Comment:
I'm not 100% sold on this change. I think it is also valuable to have a concise README. Let's see how it goes :)